<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonOrderForm</class>
 <widget class="QDialog" name="CommonOrderForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Общий шаблон ордер</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxContract">
       <property name="title">
        <string>Ордер</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Номер договора</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="pushButtonAddShipper">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="QLabel" name="labelDopolnit">
            <property name="text">
             <string>Дополнительно</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditnumberContract"/>
          </item>
          <item row="2" column="1">
           <widget class="ComboBoxShippers" name="comboBoxShippers"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Водитель</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="ComboBoxDrivers" name="comboBoxDriver"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Заказчик</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Дата</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QDateEdit" name="dateEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="showGroupSeparator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="calendarPopup">
             <bool>false</bool>
            </property>
            <property name="date">
             <date>
              <year>2000</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButtonAddDriver">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButtonCreateFrom">
            <property name="text">
             <string>Создать из</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QGroupBox" name="groupBoxNotes">
       <property name="title">
        <string>Заметки</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEditNote"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBoxRoute">
       <property name="title">
        <string>Маршрут</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Маршрут</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Ставка</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Доезд</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="spinBoxArrival">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spinBoxRoute">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelAnte">
            <property name="text">
             <string>0,00</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="groupBoxDocuments">
       <property name="title">
        <string>Документы</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Период отправки</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>2 копии ЦМР</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Оригинал договора</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="ComboBoxPeriodSendMail" name="comboBoxPostPeriad">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox2CopyCMR">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxOriginalContract">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBoxPayment">
       <property name="title">
        <string>Оплата</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Валюта</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Период оплаты</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Цена</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBoxPaymentPeriod" name="comboBoxPaymentPeriod"/>
          </item>
          <item row="1" column="2">
           <widget class="ComboBoxCurrency" name="comboBoxCurrency"/>
          </item>
          <item row="1" column="0">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxCost">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <double>99999999.989999994635582</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.010000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBoxPaymentPeriod</class>
   <extends>QComboBox</extends>
   <header>comboboxpaymentperiod.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxCurrency</class>
   <extends>QComboBox</extends>
   <header>comboboxcurrency.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxPeriodSendMail</class>
   <extends>QComboBox</extends>
   <header>comboboxperiodsendmail.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxShippers</class>
   <extends>QComboBox</extends>
   <header>comboboxshippers.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBoxDrivers</class>
   <extends>QComboBox</extends>
   <header>comboboxdrivers.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CommonOrderForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CommonOrderForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxArrival</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CommonOrderForm</receiver>
   <slot>slotArrivalChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>doubleSpinBoxCost</sender>
   <signal>valueChanged(double)</signal>
   <receiver>CommonOrderForm</receiver>
   <slot>slotCostChanged(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxRoute</sender>
   <signal>valueChanged(int)</signal>
   <receiver>CommonOrderForm</receiver>
   <slot>slotRouteChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotArrivalChanged(int)</slot>
  <slot>slotCostChanged(double)</slot>
  <slot>slotRouteChanged(int)</slot>
 </slots>
</ui>
